packages/Molecules/Markup/examplePlugins.js

```diff
@@ -0,0 +1,93 @@
+
+const pluginButton = function () {
```

Comment:
As your storybook demo is not demo all these example plugins, maybe do some cleanup here.